docker-compose.yml

```diff
@@ -216,21 +216,21 @@ services:
   # Registries #
   ##############
 
-  # registries:
-  #   # image: quay.io/centerforopenscience/ember-preprints:develop-development-local
-  #   build: ../ember-osf-registries
-  #   command: /bin/bash -c "yarn --pure-lockfile --no-progress --no-emoji && ./node_modules/bower/bin/bower install --allow-root --config.interactive=false && ./node_modules/ember-cli/bin/ember serve --host 0.0.0.0 --port 4300"
-  #   restart: unless-stopped
-  #   environment:
-  #     - BACKEND=local
-  #   expose:
-  #     - 4300
-  #   ports:
-  #     - 4300:4300
-  #   volumes:
-  #     - registries_dist_vol:/code/dist
-  #   stdin_open: true
-  #   tty: true
+  registries:
+    image: quay.io/centerforopenscience/ember-osf-registries:develop-development-local
+    build: ../ember-osf-registries
```

Comment:
This line needs to be removed as it defaulted to my local ember-osf-registries instead of the image